requirements.txt

```diff
@@ -1,4 +1,4 @@
-Jinja2==2.10.1
+Jinja2==2.11.3
```

Comment:
Do we need the exact version, or should this be `>=` ?

Reply:
Probably `~=` would be even better

Reply:
but that limits us to the 2.11 release - I guess the 2.12 release would work as well, probably even the 3.0 release, as we don't use any advanced jinja features (I would assume)
Or switch to `~=2.11` - after all we are not affected by the vulnerability ...

Reply:
Yeah, that's even better. I'll merge this to make the dependabot shut up, but we should fix it like you suggest in next